torch_np/tests/test_basic.py

```diff
@@ -27,6 +27,8 @@
 
 
 one_arg_funcs += [getattr(w, name) for name in _unary_ufuncs.__all__]
+one_arg_funcs = one_arg_funcs[:-1]   # FIXME: remove np.invert
```

Comment:
aren't you actually removing `positive` here? If you want to remove `invert, do a `if name not "invert` in the ist comprehension above.

Reply:
Removing this line leads to `RuntimeError: "bitwise_not_cpu" not implemented for 'Double'`, so it certainly is np.invert.

There's a temporary technical glitch however: 

```
(Pdb) p one_arg_funcs[-1]
<function deco_unary_ufunc.<locals>.wrapped at 0x7f5a96853790>
```
Let me keep this hack as is for now (note the FIXME comment) --- until a refactor that would hopefully remove the need for two-step wrapping of ufuncs.

Reply:
fine